src/components/App.js

```diff
@@ -0,0 +1,13 @@
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+function App() {
+  return (
+    <>
+      <Display />
+      <ButtonPanel />
+    </>
+  );
+}
```

Comment:
## Enforce the use of ES6 arrow syntax during function declaration
- [ ] Please implement this `App` component using ES6 arrow component. More on why this is recommended [here](https://codeutopia.net/blog/2015/01/06/es6-what-are-the-benefits-of-the-new-features-in-practice/#:~:text=Arrow%20functions,simpler%20code%20in%20some%20cases).
```suggestion
const App = () => {
  return (
    <>
      <Display />
      <ButtonPanel />
    </>
  );
}
```